src/_Classics_/caesar_cipher/caeserCipher.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
Creating a completely new file is not acceptable. I am marking this PR as Invalid and closing it. However, we can go ahead and keep your Unit Tests somehow

Reply:
Hey give me sometime as I have university exams going on, I'll try to update the old code itself asap.

Reply:
But I am not clear as to why a new code (that solves previous issues and have some new features) is not acceptable. Would you not consider this?